### ISSUE TITLE
Installation: Deprecation warning about @graphql-toolkit/common@0.10.7 dependency suggesting GraphQL Toolkit is deprecated

### ISSUE THREAD

**Issue by azuxx** [dotansimha/graphql-code-generator#4211]
Hello, here's some warning related to your cli when doing yarn install for my react native app.

**Describe the bug**

```
[1/4] 🔍  Resolving packages...
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/core > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/graphql-file-loader > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/json-file-loader > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/schema-merging > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @graphql-codegen/cli > graphql-config > @graphql-toolkit/url-loader > @graphql-toolkit/common@0.10.7: GraphQL Toolkit is deprecated and merged into GraphQL Tools, so it will no longer get updates. Use GraphQL Tools instead to stay up-to-date! Check out https://www.graphql-tools.com/docs/migration-from-toolkit for migration and https://the-guild.dev/blog/graphql-tools-v6 for new changes.
warning @types/graphql@14.5.0: This is a stub types definition. graphql provides its own type definitions, so you do not need this installed.
```

**To Reproduce**

package.json of the react native app.

```
//...
"@graphql-codegen/fragment-matcher": "^1.15.2",
"@graphql-codegen/introspection": "^1.15.2",
"@graphql-codegen/typescript": "^1.15.2",
"@graphql-codegen/typescript-operations": "^1.15.2",
"@graphql-codegen/typescript-react-apollo": "^1.15.2",
//...
```

How can I fix this issue?
Thanks,

**Comment by dotansimha** [comment]
Thanks @azuxx ! 
It seems like an outdated version in `graphql-config` that causes that, because GraphQL Codegen is up-to-date and uses new `@graphql-tools/`.
In terms of compatibility, no need to worry, we deprecated `@graphql-toolkit/` packages and just moved everything with the same code to `@graphql-tools/` scope. 

@kamilkisiela I think you should update `graphql-config` to use latest `@graphql-tools` ;)

**Comment by dotansimha** [comment]
Fixed in `v1.16.0`
